### ISSUE TITLE
PDFUA-247: Tab order for pages with annotations missing

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-4#247]
Jira issue originally created by user @PaulRayius:

This example demonstrates a failure condition in which the document's [tab order](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#tab-order) for [annotations](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#annotation), such as form fields, links or notes, is not set. 

Note: In the sample file, the Tab order is set to “Unspecified.” However, in Acrobat, there is a setting (in the Accessibility preferences) for “Use document structure for [tab order](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#tab-order) when no explicit [tab order](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#tab-order) is specified.” If that checkbox is checked, the sample file may display a correct [tab order](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#tab-order) by chance.

**Comment by zakkinsey** [comment]
Comment created by @unknown-jira-user-8:

Arlington error:

name tree Names object was missing when Kids was also missing for Trailer (as XRefStream)-Root (as Catalog)-StructTreeRoot-IDTree (as name-tree)

**Comment by zakkinsey** [comment]
Comment created by @unknown-jira-user-8:

Fixed by removing IDTree

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

Replaced file with one with embedded fonts.  Note:  I needed to change the font of "Submit" from Helvetica Bold to just Helvetica.  (I also had to do this in #245 but, for some reason, not the pass case example.  I hope this is ok.)

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

Note: (Same comment as in #245)  I checked with Acrobat and, in addition to the Tab order fail, Acrobat fails this due to "improper use of Lbl" (as a child of P, not in a list).  

CommonLook flags this, too, although CommonLook will be changing this functionality, soon.
